kafka-connector-demo/processes/Demo.p.json

```diff
@@ -0,0 +1,171 @@
+{
+  "format" : "10.0.0",
+  "id" : "18833AFFFC2AC1DD",
+  "config" : {
+    "data" : "com.axonivy.connector.kafka.demo.DemoData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "RequestStart",
+      "name" : "start.ivp",

```

Comment:
please tag this one as 'demo' ... so that it will be automatically opened after installing this connector